### DISCUSSION TITLE
[SOLVED] Documentation for modification of messages' score property

### DISCUSSION THREAD

**Discussion by rubisco57** [martinrotter/rssguard#738] [Q&A]
### Brief description of the feature request

RSS Guard's ability to label and score messages attracted my attention. There might be an obvious way, but I could not find out how to edit a message's score from within the application by reading the [documentation](https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#user-content-messageobject-class).
If someone knows how to do this, please let me know ;)

**Comment by martinrotter** [comment]
```js
function filterMessage() {
  msg.score = 63.5; // value between 0.0 - 100.0
  return MessageObject.Accept;
}
```

Of course the logic for evaluating each message must be according your needs. You simply assign the number to the "score" property and accept message. That's it.

Typical usage could be that you find some keywords in the message and depending on how those keywords are important, you assign the score.

**Comment by martinrotter** [comment]
Converting this to discussion.
